DirectEmployers/MyJobs
# [PD-2381] Fixed contact form

# Testing
1. Add a [contact to a partner](http://qc.secure.my.jobs/prm/view/details/edit?partner=20525&ct=25071). You'll notice that in inspector, you get a 500 response.
2. Repeat those steps with this branch, and all should be well.

I've run the regression test on qc and this branch:
![2016-05-10-092624_956x1056_scrot](https://cloud.githubusercontent.com/assets/93686/15148053/97b850c0-1691-11e6-8bca-bd18574e5521.png)

The last test status is on this branch, while the ones before was on QC.
